src/boot/measure.c

```diff
@@ -169,29 +169,42 @@ static EFI_TCG2_PROTOCOL *tcg2_interface_check(void) {
             capability.StructureVersion.Minor == 0) {
                 EFI_TCG_BOOT_SERVICE_CAPABILITY *caps_1_0 =
                         (EFI_TCG_BOOT_SERVICE_CAPABILITY*) &capability;
```

Comment:
hmm, this better be solved by a union tbh

Reply:
pre-existing condition since 2017

Reply:
yeah, but if you touch it let's fix it

Reply:
This needs to be backported, so better to leave that for later

Reply:
well, fixing this properly doesn't make backports impossible, you know

Reply:
the crash being fixed is unrelated to this